src/snosearch/queries.py

```diff
@@ -1143,21 +1143,13 @@ class BasicReportQueryFactory(BasicSearchQueryFactory):
     def __init__(self, params_parser, *args, **kwargs):
         super().__init__(params_parser, *args, **kwargs)
 
-    @assert_one_returned(error_message='Report view requires specifying a single type:')
+    @assert_something_returned(error_message='Report view requires specifying types:')
```

Comment:
Do we actually need to enforce this rule?

Reply:
Yes...